onboard/nucleo_bridge/Frontend.h

```diff
@@ -0,0 +1,197 @@
+#ifndef FRONTEND_H
+#define FRONTEND_H
+
+//#define MULTITHREADING
+
+#include "Controller.h"
+#include "Hardware.h"
+
+#include <unordered_map>
+#include <string>
+#include <lcm/lcm-cpp.hpp>
+#include <thread>
+#include <chrono>
+
+#include <rover_msgs/RAOpenLoopCmd.hpp>
+#include <rover_msgs/SAOpenLoopCmd.hpp>
+#include <rover_msgs/ArmPosition.hpp>
+#include <rover_msgs/SAClosedLoopCmd.hpp>
+#include <rover_msgs/RAConfigCmd.hpp>
+#include <rover_msgs/SAConfigCmd.hpp>
+#include <rover_msgs/HandCmd.hpp>
+#include <rover_msgs/GimbalCmd.hpp>
+#include <rover_msgs/RAPosData.hpp>
+#include <rover_msgs/SAPosData.hpp>
+//#include <rover_msgs/RAZeroTrigger.hpp>
+#include <rover_msgs/SAZeroTrigger.hpp>
+#include <rover_msgs/FootCmd.hpp>
+#include <rover_msgs/ArmPosition.hpp>
+
+#define LCM_INPUT const lcm::ReceiveBuffer* receiveBuffer, const std::string &channel
+#define NOW std::chrono::high_resolution_clock::now()
+using namespace rover_msgs;
+
+class FrontEnd {
```

Comment:
Add class and function level comments throughout.

Reply:
Implemented